From berggian@gmail.com Tue Nov  8 15:06:43 2011
Date: Wed, 9 Nov 2011 00:06:41 +0100
Message-ID: <CAD_R4=32=5gaU_OhNaM_b3-uX=G_mRFkn4Hrkn9WTT=Z+GY86g@mail.gmail.com>
Subject: [PATCH] samsung-laptop.c: support for RV420/RV520/RV720/E3530/S3530/E3420/E3520
From: Gianni Bergamo <berggian@gmail.com>
To: Greg KH <gregkh@suse.de>

This patch adds support for RV420/RV520/RV720/E3530/S3530/E3420/E3520
samsung laptops. Tested on 3.0.8 kernel.

Signed-off-by: Gianni Bergamo <berggian@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/platform/x86/samsung-laptop.c |    9 +++++++++
 1 file changed, 9 insertions(+)

--- a/drivers/platform/x86/samsung-laptop.c
+++ b/drivers/platform/x86/samsung-laptop.c
@@ -780,6 +780,15 @@ static struct dmi_system_id __initdata s
 		},
 		.callback = dmi_check_cb,
 	},
+	{
+		.ident = "RV420/RV520/RV720/E3530/S3530/E3420/E3520",
+		.matches = {
+			 DMI_MATCH(DMI_SYS_VENDOR, "SAMSUNG ELECTRONICS CO., LTD."),
+			 DMI_MATCH(DMI_PRODUCT_NAME, "RV420/RV520/RV720/E3530/S3530/E3420/E3520"),
+			 DMI_MATCH(DMI_BOARD_NAME, "RV420/RV520/RV720/E3530/S3530/E3420/E3520"),
+		},
+		.callback = dmi_check_cb,
+	},
 	{ },
 };
 MODULE_DEVICE_TABLE(dmi, samsung_dmi_table);
